docs/api_v0.yml

```diff
@@ -676,7 +676,7 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Comment"
-    SiteConfig:
```

Comment:
Not sure about this, it probably shouldn't change, right @rhymes?

Reply:
We're not changing the representation so this is a safe change